style.css

```diff
@@ -144,43 +139,34 @@ main {
 }
 
 .works_heading {
-  width: 95vw;
-  height: auto;
+  width: 95%;
   color: #172b4d;
   font-family: "Inter", sans-serif;
   font-style: normal;
   font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  align-items: center;
+  font-size: 2.5rem;
+  line-height: 3.3rem;
   text-align: center;
-  gap: 24px;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  margin-bottom: 30px;
+  margin-bottom: 1.8rem;
 }
 
 .indicator {
   display: block;
-  width: 48px;
+  width: 3rem;
   height: 4px;
   border-radius: 24px;
-  flex: none;
-  flex-grow: 0;
-  order: 1;
   background-color: #36b37e;
 }
 
 .project_section {
   display: grid;
-  row-gap: 58px;
+  row-gap: 3.6rem;
 }
 
 /* Individual card styling for projects */
 
 .single_project_card {
-  width: 95vw;
+  width: 95%;
   height: 100vh;
```

Comment:
- [ ] Your project card is great, but due to the `vh` value on the `height` attribute, the cards can have a lot of whitespace on the bottom and a distorted image. I recommend you to use fixed values for the height in this case. Kindly fix that.
 - Your design:
 - 
<img width="1151" alt="image" src="https://user-images.githubusercontent.com/61919617/206434712-94b7ad2a-bed3-44ba-a861-997262f9caef.png">

 - Figma template: 
 - 
<img width="1002" alt="image" src="https://user-images.githubusercontent.com/61919617/206434799-cfc90bc5-538f-4a52-8ba2-3755b2261016.png">